CHANGELOG.md

```
@@ -0,0 +1,57 @@
+# Changelog
+
+All notable changes to the Gruvbox Crisp Color Theme Collection will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.1.8] - 2025-01-06
+
+### Added
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)